libs/template/src/lib/let/let.directive.ts

```diff
@@ -306,6 +307,8 @@ export class LetDirective<U> implements OnInit, OnDestroy {
     this.templateManager.addTemplateRef('rxSuspense', templateRef);
   }
 
+  @Output() readonly rendered = defer(() => this.renderAware.rendered$);
```

Comment:
```suggestion
  // We use defer here as the as otherwise the the `@Output` decorator subscribes earlier than the arnderAware property is assigned
  @Output() readonly rendered = defer(() => this.renderAware.rendered$);
```